gqcp/src/Mathematical/Optimization/CMakeLists.txt

```diff
@@ -1,3 +1,2 @@
-add_subdirectory(Eigenproblem)
```

Comment:
Good catch. Could you also remove the related directory from `cmake/Doxyfile.in`?